### PR TITLE
Here is my Java solution for diamond pattern

### PR DESCRIPTION
**Pull request by Bhaswati-Saha**
#22 I have done the solution of this issue in java. Please check if there is any error.

### CODE REVIEW

File: snippets/java/diamondSquare.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+
+## Input
+```
+Enter row count :
+4
+```
+
+## Output
+```
```
Comment (kaustubhgupta) [inline snippets/java/diamondSquare.md]: Put the output of this program as an image file. Refer to this directory for the implementation of the same:
https://github.com/Developer-Students-Club-MAIT/cool_code_snippets/blob/master/snippets/python/Diamond_Square/Diamond_Square.md

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: snippets/java/diamondSquare.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    * * * 
+  * * * * * 
+* * * * * * * 
+  * * * * * 
+    * * * 
+      * 
+```
+
+## Contributed By
+-----------------------------------------------------------------------------------------------------------------
```
Comment (kaustubhgupta) [inline snippets/java/diamondSquare.md]: Follow simple table format

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by Bhaswati-Saha** [comment]
#22  I have made the modifications.

**Review by kaustubhgupta — APPROVED** [review]

File: snippets/java/Diamond_Square/diamondSquare.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    }
+}
+```
+
+## Description
+
+`Scanner` is a java.util package class that is used to take input from console. 
+
+First outer for loop displays half of the diamond pattern, second outer for loop displays the remaining half of the diamond pattern.
+
```
Comment (sakshamarora1) [inline snippets/java/Diamond_Square/diamondSquare.md]: Looks great! Just move the description block above the code block and it'll be good to merge! :)

Comment (Bhaswati-Saha) [reply snippets/java/Diamond_Square/diamondSquare.md]: Done

**Review by kaustubhgupta — APPROVED** [review]

**Pull request merged**